packages/info/index.ts

```diff
@@ -5,14 +5,18 @@ import * as process from "process";
  * Prints debugging information for webpack issue reporting
  */
 
+export function information() {
+	return {
+		Binaries: ["Node", "Yarn", "npm"],
+		Browsers: ["Chrome", "Firefox", "Safari"],
+		System: ["OS", "CPU"],
+		npmGlobalPackages: ["webpack", "webpack-cli"],
+		npmPackages: "*webpack*",
+	};
+}
+
 export default async function info() {
 	process.stdout.write(
-		await envinfo.run({
-			Binaries: ["Node", "Yarn", "npm"],
-			Browsers: ["Chrome", "Firefox", "Safari"],
-			System: ["OS", "CPU"],
-			npmGlobalPackages: ["webpack", "webpack-cli"],
-			npmPackages: "*webpack*",
-		}),
+		await envinfo.run(information()),
```

Comment:
does envinfo write to stdout by default?

Reply:
I don't know

Reply:
Test it locally!

Reply:
Automatically, it doesn't